react/src/DefaultRenderBlock.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
In some of these files `React` isn't directly referenced, which implies to me that the line can be dropped entirely. Is it required for some JSX/TSX magic, or can this line be dropped? If it can't be dropped, is there a more specific export of react which triggers the magic? Bringing everything in might have a negative impact on tree shaking.

The linked codemod suggests that what you have here is correct, but asking just in case.

Reply:
jsx elements requires react to be in module scope
```
<div></div>
```
is converted into
```
React.createElement('div')
```